_posts/2021-07-01-origami-2.0-and-how-to-migrate.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Origami 2.0 announcement and guides on how to migrate products
+description: We have released Origami 2.0, which brings full NPM support and drops support for Bower. This post explains the changes and how to migrate products to the new version.
+author: Origami
+---
+
+We are have completed our migration of Origami from Bower onto the [NPM registry](https://www.npmjs.com/). The current versions which are on Bower are now deprecated and we recommend all products migrate to the new implementation to keep up-to-date with future releases to Origami.
+
+The [FT Bower Registry](https://origami-bower-registry.ft.com/) has now been placed into a maintenance only mode for 12 months, after then we will look to decommission the service completely.
+
+Moving Origami from Bower to NPM provides many benefits, such as:
+- Aligning our front-end practices with those now most commonly used across the company and the wider industry, which should help the onboarding of new software engineering hires.
+- Origami components can now make use of the 1.6 million packages which are on the NPM registry.
+- Origami components will be able to use modern tooling instead of the Origami team having to build bespoke tooling ourselves.
+- Services such as Snyk, Dependabot, and Renovate will be able to work for the Origami dependencies a project has.
+
+# What has changed for Origami users?
+
+The code for Origami components has remained largely the same, aiming to make the migration simpler for our users.
+
+There is one change, the package name for all components now start with `@financial-times/`, as they are published under the financial-times namespace on npm. This change means that when importing the javascript and/or sass of a component, you will need to prepend `@financial-times/` to the imported package name.
```

Comment:
```suggestion
The code for Origami components has remained largely the same, aiming to make the migration simpler for our users. The one change we had to make was making the package name for all components now start with `@financial-times/`, as they are published under the financial-times namespace on npm. This change means that when importing the javascript and/or sass of a component, you will need to prepend `@financial-times/` to the imported package name.

If using Origami components via NPM, it is required to use NPM version 7, this is because version 7 ensures that Origami components are only installed once and no conflicting versions are installed. Origami components do not support using older versions of NPM.
```